project/Prototypes.scala

```diff
@@ -36,7 +36,8 @@ trait Prototypes {
     cleanAll := Def.taskDyn {
       val allProjects = ScopeFilter(inAnyProject)
       clean.all(allProjects)
-    }.value
+    }.value,
+    unmanagedClasspath in Runtime += file("ui/dist")
```

Comment:
Does this mean if it doesn't exist then the app compiles fine, but then fails in runtime? If so, I don't think thats the best behaviour. 

As it stands now if you don't compile(move) the clientside assets to the correct place (running `make compile` or `make watch`) then serverside compilation fails. That makes sense to me and I would rather this behave the same